nix/default.nix

```diff
@@ -0,0 +1,19 @@
+let
+  sources = import ./sources.nix;
+in
+import sources.nixpkgs {
+  overlays = [
+    (self: super: {
+      # niv = (import sources.niv { pkgs = self; }).niv;
```

Comment:
Could uncomment this to get the latest version of niv, instead of the one in the nixpkgs snapshot